lab_1/scripts/update-pipeline-definition.sh

```
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+BRANCH="main"
+OWNER="razumovich"
+POLL_SOURCE=0
+CONFIG="development"
+
+if ! command -v jq &> /dev/null; then
+    echo "jq is not found"
+    exit
+fi
+
+read -p "Pipeline file: " PIPELINE_FILE
+
+if [[ ! $PIPELINE_FILE ]]; then
+    echo "Pipeline file has not be specified. Please specify one."
+    exit
+fi
+
+while getopts ":b:o:p:c:" OPTIONS
+do
+    case $OPTIONS in
+        b) BRANCH=$OPTARG ;;
+        o) OWNER=$OPTARG ;;
+        p) POLL_SOURCE=$OPTARG ;;
+        c) CONFIG=$OPTARG ;;
+        *) ;;
+    esac
+done
+
+curl $PIPELINE_FILE \
```

Don't see implementation for this point:

`4. The script should validate if the necessary properties are present in the given JSON definition. If not, it should throw an error and stop execution.`